Dynatrace/dynatrace-operator
# Copies agent configs from readonly csi volume to emptyDir using standalone init

# Description

Follow up of https://github.com/Dynatrace/dynatrace-operator/pull/1722

In case `feature.dynatrace.com/injection-readonly-volume` feature flag is used the `install-oneagent` init-container will move the configs from the readonly csi-volume to an emptyDir that is mounted to the app container where these configs are expected.

## How can this be tested?
Use the `feature.dynatrace.com/injection-readonly-volume` feature-flag while using a that uses the CSI driver.
Deploy/Inject  sample-apps
Shell into a sample pod and check the `/opt/dynatrace/oneagent-paas/agent/conf` path, and it should have the config files that are originally from the CSI volume. (its a good indicator if the `_ruxitagentproc.conf` is present)


## Checklist
